CMakeLists.txt

```diff
@@ -57,7 +57,6 @@ include(Utils)
 include(OSConfiguration)
 include(Options)
 include(SendFileHack)
-include(AppleConfiguration)
```

Comment:
@rafaelsamenezes AppleConfiguration is included in OSConfiguration already, so I've removed it here. Is that OK?